include/z64player.h

```diff
@@ -832,15 +842,11 @@ typedef struct Player {
     /* 0x06A8 */ Actor* unk_6A8;
     /* 0x06AC */ s8 unk_6AC;
     /* 0x06AD */ u8 unk_6AD;
-    /* 0x06AE */ u16 unk_6AE;
-    /* 0x06B0 */ s16 unk_6B0;
+    /* 0x06AE */ u16 unk_6AE_rotFlags; // See `UNK6AE_ROT_` macros. If its flag isn't set, a rot steps to 0.
+    /* 0x06B0 */ s16 upperLimbYawSecondary;
```

Comment:
What is the secondary yaw? Is `upperLimbRot.y` the primary yaw? Could you add a small comment on how they differ?

Reply:
It looks like its only meaningfully used when riding a horse. Id be fine with this temporary name (and no comment) for now until we figure it out more, i imagine the name will change